spec/features/bonds_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe 'bonds data' do
+  before(:each) do
+    login_as FactoryGirl.create(:user)
+  end
+
+  def person_with_charge(first_name, last_name, bond_amount, released: false)
```

Comment:
Can we rename this to `create_person_with_charge`, to be more explicit that the method creates DB records?